<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Time Code</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>input directory</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="ui_input_dir"/>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="ui_input_browse">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>output directory</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="ui_output_dir"/>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="ui_output_browse">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>570</width>
        <height>400</height>
       </size>
      </property>
      <property name="title">
       <string>preview</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QGraphicsView" name="ui_preview"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>settings</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>pos</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QSpinBox" name="ui_pos_x">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="prefix">
              <string>x: </string>
             </property>
             <property name="maximum">
              <number>99999</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="ui_pos_y">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="prefix">
              <string>y: </string>
             </property>
             <property name="maximum">
              <number>99999</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>fps</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="ui_framerate"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>color</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QFrame" name="ui_color">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="ui_color_browse">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>frame</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QFrame" name="ui_frame_color">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="ui_frame_color_browse">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>font</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="ui_font_name">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>52</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="text">
            <string>default</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ui_font_browse">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="ui_cancel_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ui_run_button">
        <property name="text">
         <string>run</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ui_quit_button">
        <property name="text">
         <string>quit</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>642</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>ui_input_dir</tabstop>
  <tabstop>ui_input_browse</tabstop>
  <tabstop>ui_output_dir</tabstop>
  <tabstop>ui_output_browse</tabstop>
  <tabstop>ui_pos_x</tabstop>
  <tabstop>ui_pos_y</tabstop>
  <tabstop>ui_framerate</tabstop>
  <tabstop>ui_font_browse</tabstop>
  <tabstop>ui_run_button</tabstop>
  <tabstop>ui_quit_button</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>ui_quit_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>374</x>
     <y>666</y>
    </hint>
    <hint type="destinationlabel">
     <x>512</x>
     <y>663</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ui_input_browse</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>browseInputDir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>629</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>597</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ui_output_browse</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>browseOutputDir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>629</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>597</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ui_font_browse</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>changeFont()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>561</x>
     <y>606</y>
    </hint>
    <hint type="destinationlabel">
     <x>584</x>
     <y>648</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ui_color_browse</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>changeColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>573</y>
    </hint>
    <hint type="destinationlabel">
     <x>596</x>
     <y>559</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ui_pos_x</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindowClass</receiver>
   <slot>updateTextPosition()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>579</y>
    </hint>
    <hint type="destinationlabel">
     <x>38</x>
     <y>673</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ui_pos_y</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindowClass</receiver>
   <slot>updateTextPosition()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>588</y>
    </hint>
    <hint type="destinationlabel">
     <x>146</x>
     <y>679</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ui_run_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>process()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>666</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>688</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ui_cancel_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>setStopFlag()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>672</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>687</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ui_quit_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>setStopFlag()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>385</x>
     <y>675</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>678</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ui_frame_color_browse</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>changeFrameColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>404</x>
     <y>611</y>
    </hint>
    <hint type="destinationlabel">
     <x>443</x>
     <y>650</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MainWindowClass</sender>
   <signal>destroyed()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>setStopFlag()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>30</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>79</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>browseInputDir()</slot>
  <slot>browseOutputDir()</slot>
  <slot>changeFont()</slot>
  <slot>changeColor()</slot>
  <slot>updateTextPosition()</slot>
  <slot>process()</slot>
  <slot>setStopFlag()</slot>
  <slot>changeFrameColor()</slot>
 </slots>
</ui>
